san/batch.py

```diff
@@ -21,11 +22,12 @@ def __create_batched_query_string(self):
         batched_queries = []
 
         for idx, query in enumerate(self.queries):
-            [metric, slug] = query[0].split('/')
+            [metric, _separator, slug] = query[0].partition('/')
             if metric in V2_METRIC_QUERIES:
-                batched_queries.append(
-                    san.sanbase_graphql.get_metric(
-                        idx, metric, slug, **query[1]))
+                if slug != '':
+                    batched_queries.append(san.sanbase_graphql.get_metric(idx, metric, slug, **query[1]))
+                else:
+                    raise SanError('Invalid metric!')
             else:
```

Comment:
why we check if slug is not empty only in the case of v2 metric?

Reply:
We check it also in get_gql_query, the checking is just in the san.query